spec/factories/clients.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :client do
+    email { Faker::Internet.email }
```

Comment:
@rakvium Done. Added validation tests for all models except Admin. For some reason build(:admin) raises " NoMethodError: undefined method `encrypted_password=' for #<Admin:0x000000033dda80>"